prereise/gather/griddata/hifld/orchestration.py

```diff
@@ -6,13 +6,18 @@
 from prereise.gather.griddata.hifld.const import powersimdata_column_defaults
 from prereise.gather.griddata.hifld.data_process.demand import assign_demand_to_buses
 from prereise.gather.griddata.hifld.data_process.generators import build_plant
+from prereise.gather.griddata.hifld.data_process.profiles import build_solar
 from prereise.gather.griddata.hifld.data_process.transmission import build_transmission
 
 
-def create_csvs(output_folder):
+def create_csvs(output_folder, nrel_email, nrel_api_key, solar_kwargs={}):
```

Comment:
From the user perspective, do we want to enable the user to generate the grid without generating the profiles? If so, we could add a flag input, and add default `None` values for the profile-specific input parameters.

Reply:
Alternatively, we may want the user to be able to create the full Grid CSVs, and then use these for multiple years. We could break `create_csvs` into a grid-step and a profiles step, or allow the user to pass a list of years to have everything handled automatically.

Reply:
Agree it makes sense to be able to create the grid and profiles separately. Assuming this isn't something that would be used often, I might leave it at something like `create_grid` and `create_profiles`, or make `create_csvs` a wrapper around those if it's convenient.

Reply:
Done.